relay-event-schema/src/protocol/types.rs

```diff
@@ -22,7 +22,7 @@ use crate::processor::{
     process_value, ProcessValue, ProcessingResult, ProcessingState, Processor, ValueType,
 };
 
-/// A array like wrapper used in various places.
+/// An array like wrapper used in various places.
```

Comment:
```suggestion
/// An array-like wrapper used in various places.
```  
my most pedantic comment yet